tests/testthat/test-format_time.R

```diff
@@ -1,6 +1,149 @@
 context("format_time")
 
+library("tibble")
```

Comment:
I'd rather avoid adding the tibble dependency just for `tribble()` in the tests. Maybe use a small `.csv` file if you think it helps improving maintainability?

Reply:
done in 7442b8a